src/components/Payment/Payment.tsx

```diff
@@ -149,6 +253,38 @@ const Payment = ({
               <Button variant="contained" color="primary" label={t('user:payment.complete_subscription')} onClick={onCompleteSubscriptionClick} />
             </React.Fragment>
           )}
+          {isChangingOffer && (
+            <div className={styles.changePlanContainer}>
+              {offers
+                .filter((o) => o.planSwitchEnabled)
+                .map((offer) => (
+                  <OfferSwitch
+                    key={offer.offerId}
+                    isCurrentOffer={offer.offerId === activeSubscription?.accessFeeId}
+                    pendingDowngradeOfferId={pendingDowngradeOfferId}
+                    offer={offer}
+                    selected={{ value: selectedOfferId === offer.offerId, set: setSelectedOfferId }}
+                  />
+                ))}
```

Comment:
Perhaps the `useOffers` can return a `pendingOffers` collection that is passed to this component.

Reply:
This logic can also be moved to the `PaymentContainer` instead of the `useOffers` hook. Although, we might need it elsewhere...